trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/testing/harness/TrinoAwsProxyTestCommonModules.java

```diff
@@ -79,7 +79,7 @@ public TestingTrinoAwsProxyServer.Builder filter(TestingTrinoAwsProxyServer.Buil
         }
     }
 
-    public static class WithAllContainers
+    public static class WithStandardContainers
```

Comment:
`WithDefaultContainers`?

I wouldn't say that `PySpark` is necessarily a "standard" container for someone using S3

Reply:
I reworked this. PTAL